uperf-wrapper/redis_utils.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/python
```

Comment:
can you please move this to https://github.com/cloud-bulldozer/snafu/tree/master/utils/redis_scripts >?

Reply:
none of the import statement worked, any ideas? I know @rsevilla87 mentioned absolute imports but no luck there as well. Will try again